packages/tasks/src/tasks/zero-shot-object-detection/data.ts

```diff
@@ -0,0 +1,61 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "object-detection-input.jpg",
+				type: "img",
+			},
+            {
+				label: "Classes",
+				content: "cat, dog, bird",
+				type: "text",
+			},
+		],
+		outputs: [
+			{
+				filename: "object-detection-output.jpg",
+				type: "img",
+			},
+		],
+	},
+	metrics: [
+		{
+			description:
+				"The Average Precision (AP) metric is the Area Under the PR Curve (AUC-PR). It is calculated for each class separately",
+			id: "Average Precision",
+		},
+		{
+			description: "The Mean Average Precision (mAP) metric is the overall average of the AP values",
+			id: "Mean Average Precision",
+		},
+		{
+			description:
+				"The APα metric is the Average Precision at the IoU threshold of a α value, for example, AP50 and AP75",
+			id: "APα",
+		},
+	],
+	models: [
+		{
+			description: "Solid zero-shot object detection model that uses CLIP as backbone.",
+			id: "google/owlvit-base-patch32",
+		},
+		{
+			description: "The improved version of the owlvit model.",
+			id: "google/owlv2-base-patch16-ensemble",
+		},
+	],
+	spaces: [
+
+	],
+	summary:
+		"Zero-shot object detection models allow users to identify objects without predefined classes. Zero-shot object detection models receive an image as input, as well as a list of candidate classes, and output the bounding boxes and labels where the objects have been detected. Zero-shot object detection is a computer vision task to detect objects and its class in images, without any prior training or knowledge of the classes. Zero shot object detection works by transferring knowledge learnt during training of one model, to classify novel classes that was not present in the training data.",
```

Comment:
```suggestion
		"Zero-shot object detection is a computer vision task to detect objects and their classes in images, without any prior training or knowledge of the classes. Zero-shot object detection models receive an image as input, as well as a list of candidate classes, and output the bounding boxes and labels where the objects have been detected.",
```

I found it a bit redundant after the two versions of the description were concatenated, so tried to simplify it a bit. Feel free to add more details if you think they are important.